src/orquestra/sdk/secrets/_auth.py

```diff
@@ -0,0 +1,63 @@
+################################################################################
+# © Copyright 2023 Zapata Computing Inc.
+################################################################################
+import os
+import typing as t
+from pathlib import Path
+
+from orquestra.sdk import exceptions
+from orquestra.sdk.schema.configs import ConfigName
+
+from .._base import _config
+from ._client import SecretsClient
+
+# env var name
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏This comment isn't too useful atm. What env var name? What is it used for?

Reply:
I agree. I've reworded it in 91e28e9, can you give it a look?